servers/rendering/renderer_rd/renderer_storage_rd.cpp

```diff
@@ -10033,7 +10033,7 @@ RendererStorageRD::RendererStorageRD() {
 		actions.renames["RESTART"] = "restart";
 		actions.renames["CUSTOM"] = "PARTICLE.custom";
 		actions.renames["TRANSFORM"] = "PARTICLE.xform";
-		actions.renames["TIME"] = "FRAME.time";
+		actions.renames["TIME"] = "FRAME_TIME";
```

Comment:
```suggestion
		actions.renames["TIME"] = "frame_history.data[0].time";
```

I think it is simpler to just use frame_history.data[0].time directly.

Reply:
Done